WebService.Test/v1/Controllers/DevicePropertiesControllerTest.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Controllers;
+using Moq;
+using WebService.Test.helpers;
+using Xunit;
+
+namespace WebService.Test.v1.Controllers
+{
+    public class DevicePropertiesControllerTest
+    {
+        private readonly Mock<IDeviceModels> deviceModelsService;
+        private readonly DevicePropertiesController target;
+
+        public DevicePropertiesControllerTest()
+        {
+            this.deviceModelsService = new Mock<IDeviceModels>();
+            this.target = new DevicePropertiesController(this.deviceModelsService.Object);
+        }
+
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
+        public void ItReturnsTheListOfPropertyNames()
+        {
+            // Arrange
+            var properties = new List<DeviceProperty>
+            {
+                new DeviceProperty {Id = "Type"},
```

Comment:
code style: there should be a space before/after, right?
```
new DeviceProperty { Id = "Type" },
```

Reply:
Done.